web/src/components/MemoView.tsx

```diff
@@ -163,6 +163,10 @@ const MemoView: React.FC<Props> = (props: Props) => {
         showPreviewImageDialog([imgUrl], 0);
       }
     }
+
+    if (e.detail === 2) {
```

Comment:
You could use a timeout to ensure a double-click is recognized correctly, even if clicks are slightly close together.

Reply:
Thank you for your suggestion!